scripts/find-updates.sh

```diff
@@ -0,0 +1,132 @@
+#!/bin/bash
+
+set -euo pipefail
+# set -x
+
+# referenced from https://gist.github.com/lukechilds/a83e1d7127b78fef38c2914c4ececc3c
+function get_latest_release() {
+  version=$(curl --silent "https://api.github.com/repos/$1/releases/latest" |
+    grep '"tag_name":' |
+    sed -E 's/.*"([^"]+)".*/\1/')
+}
+
+workdir="${HOME}/work/lokomotive"
+format="%-20s %18s %18s\n"
+printf "${format}" "Component" "Current Version" "Latest Version"
+printf "${format}" "---------" "---------------" "--------------"
+
+###########################
+# k8s
+current_version=$(grep 'k8s.gcr.io/kube-apiserver' assets/lokomotive-kubernetes/bootkube/variables.tf | cut -d":" -f2)
+
+get_latest_release kubernetes/kubernetes
+printf "${format}" "kubernetes" "${current_version}" "${version}"
+
+###########################
+# calico
+current_version=$(grep 'calico/node' assets/lokomotive-kubernetes/bootkube/variables.tf | cut -d":" -f2)
+
+get_latest_release projectcalico/calico
+printf "${format}" "calico" "${current_version}" "${version}"
+
+###########################
+# etcd
+current_version=$(grep 'ETCD_IMAGE_TAG=' assets/lokomotive-kubernetes/aws/flatcar-linux/kubernetes/cl/controller.yaml.tmpl | cut -d"=" -f3)
+
+get_latest_release etcd-io/etcd
+printf "${format}" "etcd" "${current_version}" "${version}"
+
+###########################
+# cert-manager
+
+cd "${workdir}"
+current_version=$(grep appVersion assets/components/cert-manager/manifests/Chart.yaml | cut -d":" -f2)
+
+tmpdir="/tmp/$RANDOM"
+mkdir "${tmpdir}"
+cd "${tmpdir}"
+helm repo add jetstack https://charts.jetstack.io >/dev/null 2>&1
```

Comment:
Would be nice to use some temporary helm home to avoid messing up with users repositories. So overriding `HELM_REPOSITORY_CONFIG` and `HELM_REPOSITORY_CACHE` would work I think. Or maybe try with overriding `HOME` ?

Reply:
Those were different vars they start with `XDG-*`.

Reply:
Maybe we can run it inside Docker then? I don't like that script will change my repositories.